src/hooks/useWalletConnect.tsx

```diff
@@ -147,14 +144,8 @@ export function useWalletConnect(): WalletConnect {
     const session = sessions[topic]
     const dAppName = session?.peer.metadata.name ?? lstrings.wc_smartcontract_dapp
 
-    try {
-      await runWithTimeout(client.disconnectSession({ topic, reason: getSdkError('USER_DISCONNECTED') }), 10000)
-    } catch (e) {
-      // In testing, this method is pretty unreliable so we can at least remove it locally.
-      console.log('walletConnect disconnectSession error', String(e))
-      client.core.relayer.subscriber.topicMap.delete(topic)
-    }
-    userDeletedSessions.add(topic)
+    // In testing, this method is pretty unreliable. May be worth replacing with something more manual.
+    await runWithTimeout(client.disconnectSession({ topic, reason: getSdkError('USER_DISCONNECTED') }), 10000)
```

Comment:
Maybe increase the timeout to 20s as in the previous commit.

Reply:
In my testing, the dapp responds pretty quickly but the sdk hangs for an unknown reason. I'd rather it fail sooner so the user can initiate it again if they want.